API/Entities/User.cs

```diff
@@ -28,6 +28,8 @@ public class User
     [Column("updated", TypeName = "timestamp with time zone")]
     public DateTime? Updated { get; set; }
 
+    public RateLimitOverrides? RateLimitOverrides { get; set; }
```

Comment:
Document that the column name is defined in `OtrContext`

Reply:
I considered leaving the column notation but thought that wouldn't make sense considering the mapping ignores it anyway. Will leave a comment